server/rest/src/test/java/org/infinispan/rest/Http2Test.java

```diff
@@ -41,6 +41,7 @@
  *
  * @author Sebastian Łaskawiec
  */
+@Test(groups = "functional", testName = "rest.Http2Test")
```

Comment:
This is silly... why TestNG doesn't pick up classes ending with `test`, just like JUnit? :imp: